generated/server/mock_controller.go

```diff
@@ -0,0 +1,64 @@
+// Automatically generated by MockGen. DO NOT EDIT!
+// Source: interface.go
+
+package server
```

Comment:
hmm, would it make sense to move this to the models or client package to avoid having to import the server package? maybe we can re-visit when we have a separate client interface?